packages/core/src/model.ts

```diff
@@ -198,13 +198,12 @@ export class Model implements Storage {
   /**
    * Construct a new proper document with all desired fields.
    * @param _class
-   * @param _id
    * @param layout
    */
-  public newDoc<T extends Doc> (_class: Ref<Class<T>>, _id: Ref<Doc>, layout: AnyLayout): T {
-    const doc = (this.assign({}, _class, layout) as unknown) as T
-    doc._id = _id
-    return doc
+  public createDocument<T extends Doc> (_class: Ref<Class<T>>, layout: Partial<T>): T {
```

Comment:
Why layout is `Parial<T>` and not `Omit<T, keyof Doc>`?

Reply:
we need to specify _space, probable we should just omit _id, _class

Reply:
Introduce DocumentValue<T>